src/types/Book.ts

```diff
@@ -2,6 +2,7 @@ export interface Book {
   id: string;
   googleId: string;
   title: string;
+  description: string;
```

Comment:
is the description the reason for recommending the book or the description of the book itself from the google api?

Reply:
Description of book itself from Books API!